### PR TITLE
Corrected & standardise documentation

### LINKED ISSUE #194: [PE-D] FAQ is a little too short
FAQ section in the User Guide could contain more questions.

 It could include more questions that users likely to ask. Such as "Is my data backed up to the internet"  

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: GeNiaaz/ped#1

### LINKED ISSUE #196: [PE-D] cadd command: address field (ug vs app)
cadd, or new client command takes in phone, email, address, and tags in the application

However, in the User guide, the address is not taken in as input

as well as 

## Suggestion

I would just like to suggest that instead, you guys remove the address altogether. I believe it adds an unnecessary layer of complexity.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: GeNiaaz/ped#3

### LINKED ISSUE #204: [PE-D] Inconsistency in table of contents (List word)
Small typo I think?

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: leeyorktat/ped#2

### LINKED ISSUE #206: [PE-D] Inconsistency in the using of images and figures
Some parts of the UG refer to the figure below while some refer to the images below. Can considering standardising the terms?

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: leeyorktat/ped#4

### LINKED ISSUE #207: [PE-D] Inconsistency in lowercase and uppercase
The examples highlighted are uppercase while the examples below are lower case. Can consider changing one to be consistent?

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: leeyorktat/ped#5

### LINKED ISSUE #210: [PE-D] Suggestion to make instructions more explicit
Can consider being more explicit, "storing their profile photo in a new folder named images within the data folder", more user-friendly to people not familiar with tech? Just a suggestion

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: leeyorktat/ped#8

### PR DESCRIPTION
**Pull request by maguireong**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -670,6 +670,8 @@ Examples:
 **Q**: I would like to contribute to this project. May I know who do I reach out to?  
 **A**: You may reach our PR Department email [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ).
 
+**Q**: Is my data backed up to the internet?  
+**A**: No, your data in FitEgo are saved in the hard disk. You will have to transfer the file, which contains your data, to whichever device you wish to continue using FitEgo on.
```
Comment (benclmnt) [inline docs/UserGuide.md:709]: Perhaps it is better to say "saved locally in your computer."?

On a side note, i think we have this somewhere on top, near quick start section.

Comment (maguireong) [reply docs/UserGuide.md:709]: oo so should I just linked it there or actually is this question redundant?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -345,14 +345,14 @@ a tag with a dash.
 
 :star: **Feature:**
 
-You can add profile picture to your client by storing their photo in `data/images` folder. You should name the photo 
+You can add a profile picture to your client by storing their photo in a new folder named images within the data folder. You should name the photo
```
Comment (dhafinrazaq) [inline docs/UserGuide.md]: not sure about this, but should we wrap the folder name (images and data) in a `code` format or some quote?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -638,11 +638,11 @@ Points to take note when editing a schedule's details:
 
 Examples:
 
-*  `schedit c/1 s/1 us/2` - Reschedules the first client in the Client List to the second session in the Session List while keeping all other fields the same
-*  `schedit c/1 s/1 pd/paid` - Indicates that the first client in the Client List has paid for the second session in the Session List while keeping all other fields the same
-*  `schedit c/1 s/1 r/did 5 pushups` - Edits the schedule containing client index 1 and session index 1 to have remark of doing 5 pushups while keeping all other fields the same
-*  `schedit c/1 s/1 w/70` - Edits the schedule containing client index 1 and session index 1 to a weight of 70kg while keeping all other fields the same
-* `schedit c/1 s/1 r/` - Clears the schedule containing client index 1 and session index 1 remarks while keeping all other fields the same
+*  `schedit c/1 s/1 us/2` reschedules the first client in the Client List to the second session in the Session List while keeping all other fields the same
+*  `schedit c/1 s/1 pd/paid` indicates that the first client in the Client List has paid for the second session in the Session List while keeping all other fields the same
```
Comment (dhafinrazaq) [inline docs/UserGuide.md:677]: probably I mistyped here, could you change this into "the first session in"...

File: docs/UserGuide.md (hunk 2 above)
Comment (maguireong) [reply docs/UserGuide.md]: it was in a code format, the guy suggested this instead and I think this will be more readable?

Comment (kelvinvin) [reply docs/UserGuide.md]: I think what Dhafin meant was:
"..in a new folder named `images` within the `data` folder.."

**Review by kelvinvin — COMMENTED** [review]
Looks good to merge

**Pull request merged**